docs/section4/4-4.md

```diff
@@ -0,0 +1,36 @@
+# ASHVAC – Rule 4-4
+
+**Rule ID:** 4-4
+ 
+**Rule Description:** Where no heating and/or cooling system is to be installed, and a heating or cooling system is being simulated only to meet the requirements described in Table G3.1, heating and/or cooling system fans shall not be simulated as running continuously during occupied hours but shall be cycled ON and OFF to meet heating and cooling loads during all hours.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules exception #1 for the proposed building  
+**Appendix G Section Reference:**  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** 
+
+1. Only applies to HVAC systems that are being simulated only to meet the requirements described in Table G3.1.
+
+**Manual Check:** None  
+**Evaluation Context:** Evaluate each data element  
+**Data Lookup:** None  
+**Function Call:** None
+
+## Rule Logic:
+
+- For each HVAC system in P_RMR: `for hvac_p in P_RMR...heating_ventilation_air_conditioning_systems:`
+    - Get HVAC system status: `status_type_p = hvac_p.status_type:`
+        - Check if HVAC system status is yet to be designed: `if status_type_p not in [NEW,EXISTING]:`
+            - Set applicability flag: `rule_applicability_check = TRUE`
+                - For each fan system in each HVAC system that meets the applicabliity check: `fan_systems_p in heating_ventilation_air_conditioning_systems_p.fan_systems:`
+                        -   Get fan system unoccupied operation control: `operation_during_occupied_p = fans_systems_p.operation_during_occupied`
+
```

Comment:
I beleive the RDS needs to be expanded to cover both occupied and unoccupied hours.

Reply:
No changes made. Decided to keep separate on team call.